lms/djangoapps/courseware/tests/test_access.py

```diff
@@ -146,29 +152,29 @@ def test__has_access_descriptor_staff_lock(self):
 
         # No start date, staff lock on
         mock_unit.visible_to_staff_only = True
-        self.verify_access(mock_unit, False)
+        self.verify_access(mock_unit, access_response.VisibilityError())
 
         # No start date, staff lock off.
         mock_unit.visible_to_staff_only = False
-        self.verify_access(mock_unit, True)
+        self.verify_access(mock_unit, self.ACCESS_GRANTED)
```

Comment:
In order to verify that existing callers of `has_access` will continue to work, we should not have to update the existing test cases for `has_access`.